### PR TITLE
Control Validation (Bug Fix + Tags Inclusion)

### PR DESCRIPTION
**Pull request by abhaydaga**
## Description
</br>
 Automation webhook controls bug fixes along with inclusion of tags for service jsons and a recommendation update in a service json.
## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Core/SVT/Services/Automation.ps1 [hunk 1]
```diff
@@ -16,8 +16,9 @@ class Automation: SVTBase
 		$webhooks = Get-AzureRmAutomationWebhook -AutomationAccountName $this.ResourceContext.ResourceName -ResourceGroupName $this.ResourceContext.ResourceGroupName -ErrorAction Stop
 		if(($webhooks|Measure-Object).Count -gt 0)
 		{
-			$controlResult.AddMessage([VerificationResult]::Verify, "Please verify below webhook(s) created for the runbooks. Remove webhook(s) which are not in use.", $webhooks)
-			$controlResult.SetStateData("Webhooks", $webhooks);                      
+			$webhookdata = $webhooks | Select-Object ResourceGroupName, AutomationAccountName, Name, Description, IsEnabled, Parameters, RunbookName, WebhookURI, HybridWorker
+			$controlResult.AddMessage([VerificationResult]::Verify, "Please verify below webhook(s) created for the runbooks. Remove webhook(s) which are not in use.", $webhookdata)
```
Comment (sbyna-msft) [inline src/AzSK/Framework/Core/SVT/Services/Automation.ps1]: can you check on how many controls are actually attested for this ? And can they be impacted with the change of object model?

File: src/AzSK/Framework/Core/SVT/Services/Automation.ps1 [hunk 2]
```diff
@@ -34,7 +35,10 @@ class Automation: SVTBase
 			$webhooks | Where-Object{$_.IsEnabled -eq $true} | ForEach-Object{
 				if(($_.ExpiryTime - $_.CreationTime).Days -gt $this.ControlSettings.Automation.WebhookValidityInDays)
```
Comment (sbyna-msft) [inline src/AzSK/Framework/Core/SVT/Services/Automation.ps1]: i think this code can be optimized. There is no need for two loops rather you can bring the if else inside the loop

Comment (abhaydaga) [reply src/AzSK/Framework/Core/SVT/Services/Automation.ps1]: Done.

File: src/AzSK/Framework/Core/SVT/Services/Automation.ps1 (hunk 1 above)
Comment (abhaydaga) [reply src/AzSK/Framework/Core/SVT/Services/Automation.ps1]: Resolved.

**Pull request merged**